docs/developer/app-store/apps/taxes.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Taxes
+sidebar_position: 2
+---
+
+# Taxes
+
+## Overview
+
+_Taxes_ is a Saleor app that allows delegating tax calculations to external tax providers. Here are the currently available providers:
+
+- [TaxJar](https://www.taxjar.com/)
+- [Avatax](https://www.avalara.com/us/en/products/calculations.html)
+- [Avatax Excise](https://developer.avalara.com/excise/) [_coming soon_]
+
+Taxes App has two views: **Channels** and **Providers**.
+
+**Channels** allows you to fill in the "ship from" address and assign a tax provider instance to the channel.
+
+**Providers** allows you to manage the tax provider instances. You can have multiple instances of the same provider.
+
+:::caution
+
+To successfully configure the Taxes App, you must have an account with one of the supported tax providers.
+
+:::
+
+## Development
+
+To run the Taxes App locally:
+
+1. Follow the [_Setup_ section in the _Development_ article](developer/app-store/development.mdx#setup).
+2. Go to the app directory.
+3. Copy the `.env.example` file to `.env`.The `.env` should contain the following variables:
+
+`SECRET_KEY` (**required**)
+
+A randomly generated key for the encryption of [Settings Manager](https://github.com/saleor/saleor-app-sdk/blob/main/docs/settings-manager.md).
```

Comment:
Add that it can be skipped on the development but its encouraged to set it. We will add more docs when we migrate SDK here

Reply:
also link sdk docs
https://github.com/saleor/saleor-app-sdk/blob/main/docs/settings-manager.md

Reply:
You mean to link the SDK docs in the `SECRET_KEY` heading? Because they are linked a line below

Reply:
Also, what happens if there is no `SECRET_KEY`? Because the SDK docs says the app will exit.